privacyidea/static/components/login/controllers/loginControllers.js

```diff
@@ -255,7 +255,7 @@ angular.module("privacyideaApp")
                     PollingAuthFactory.start($scope.check_authentication);
                 }
                 // In case of u2f we do:
-                if ($scope.u2fSignRequests) {
+                if ($scope.u2fSignRequests.length > 0) {
```

Comment:
What happens if ``$scope.u2fSignRequests`` would be undefined? (it probably will not be), but then we would have a script error.

Reply:
Hmm, it shouldn't be (because we set it to ``Array()``) in Line 214. But I guess you're right, we could go the safe route here :)

Reply:
OK, no. THis is fine. You are right. We set this to ``Array()``  and later only push to this array. So this is ok.